CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.1 (2020-05-27)
```

Comment:
If we're using SemVer, I think this would be a `MINOR` release, adding backwards compatible functionality, and not a `PATCH` bugfix.

Shouldn't this be `## 0.3.0 (...)`?

Reply:
You're right. Based on previous updates it should be 0.3.0. I've update it.